### DISCUSSION TITLE
Multivariate Data Assimilation in LIS

### DISCUSSION THREAD

**Discussion by Arijit22IITDelhi** [NASA-LIS/LISF#1492] [Q&A] [answered]
Is it possible to assimilate more than one satellite data set of different parameters simultaneously within NASA LISF (for example, MODIS LAI as well as SMAP soil moisture)?. If yes, then what necessary changes have to be made in the LDT and LIS config files while defining the data assimilation specifications?

**Comment by dmocko** [comment, marked as answer]
Hi @Arijit22IITDelhi - Yes, this is possible to do.

In LDT, you would need to generate CDFs for each product that you want to assimilate.  Note that we typically do not generate/use CDFs for LAI data assimilation.

In LIS, you just need to specify the `Number of data assimilation instances:` in the lis.config, and then have a column with entries for each product that you assimilate.  Here is an example:

https://github.com/NASA-LIS/LISF/blob/master/lis/configs/557WW-7.5-FOC/NRT_GLOBAL/lis.config.global.noahmp401.galwem17#L71-L136

**Reply by Arijit22IITDelhi** [reply]
Thanks a lot.
